solution

Stepan-TS opened the pull request:
[DEMO LINK](https://Stepan-TS.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://Stepan-TS.github.io/layout_moyo-header/report/html_report/)

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
```
`box-sizing` is not inherited and body don't have any paddings or borders by default
this rule is innecessary here

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,66 @@
+}
+
+.logo__link {
+  padding: 10px 0;
```
your header elements are aligned, no need to align them again by paddings

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,66 @@
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  font-family: "Roboto", sans-serif;
```
set font styles to `html` tag

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,66 @@
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}

```

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,66 @@
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+}
```
It should work for every link that is active. If i set this class to other elements - i'll see 
<img width="1437" alt="Screenshot 2023-01-25 at 23 59 01" src="https://user-images.githubusercontent.com/84275496/214701072-1ff45873-4aaf-402e-8ed1-1005a6a5bcc7.png">

Stepan-TS commented:
1. removed box-sizing;
2. removed padding in .logo__link;
3. seted font styles to html tag;
4. fixed margin-right in .nav__item;
5. fixed active link;

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -1 +1,63 @@
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
```
```suggestion
  background-color: #00acdc;
  border-radius: 8px;
  bottom: 0;
  left: 0;
  right: 0;
```

OleksiiBlezniuk replied:
not critical, but as for me its more logical
